src/tests/ftest/container/boundary.yaml

```diff
@@ -0,0 +1,37 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+timeout: 3500
+server_config:
+  name: daos_server
+  servers:
+    scm_size: 600
+    targets: 1
+pool:
+  name: daos_server
+  scm_size: 150M
```

Comment:
Isn't it better to use larger pool if you want to test creating pools in parallel? Creating a 150MB pool doesn't take a long time, so the overlap time may not be very long. We're creating at most 100 pools, so we're only using up to 15GB of storage. We're using HW Medium, so we should have a lot more storage.

Reply:
Since we are testing a large number of pools and containers, 150M is the appropriated size to avoid out of space -1007 issue when testing locally.
I will update the pool size with 1G.